ddtrace/settings/_config.py

```diff
@@ -512,15 +496,8 @@ def __init__(self):
 
         self._128_bit_trace_id_enabled = _get_config("DD_TRACE_128_BIT_TRACEID_GENERATION_ENABLED", True, asbool)
 
-        self._128_bit_trace_id_logging_enabled = _get_config("DD_TRACE_128_BIT_TRACEID_LOGGING_ENABLED", False, asbool)
-        if self._128_bit_trace_id_logging_enabled:
-            deprecate(
-                "Using DD_TRACE_128_BIT_TRACEID_LOGGING_ENABLED is deprecated.",
-                message="Log injection format is now configured automatically.",
-                removal_version="3.0.0",
-                category=DDTraceDeprecationWarning,
-            )
-
+        # Disabling DD_TRACE_128_BIT_TRACEID_LOGGING_ENABLED is not supported. Remove this configuration in the future.
+        self._128_bit_trace_id_logging_enabled = True
```

Comment:
cc: @mtoffl01 

Not sure if we should pull this out into it's own PR. ``DD_TRACE_128_BIT_TRACEID_LOGGING_ENABLED`` configuration is being removed so we need to enable this functionality by default. 

Maybe this could use it's own release note.